Martium.TravelInfo.App/Forms/TravelInfoForm.cs

```diff
@@ -277,6 +282,14 @@ private bool CheckIsDecimal(string text)
             return success;
         }
 
+        private void LoadCountryIsoCollections()
+        {
+            foreach (KeyValuePair<string, string> countryIso in IsoDictionary.CountryDictionary)
+            {
+                CountryIsoComboBox.Items.Add(countryIso.Key);
```

Comment:
use `value` instead of `key` so it would be more clear for user what he selects

Reply:
make departure country combo box more wider in form design so it would fit longer country names